package.json

```
@@ -4,35 +4,36 @@
   "private": true,
   "description": "Release Action for Simple Icons",
   "license": "CC0-1.0",
-  "exports": "./src/index.js",
   "type": "module",
+  "exports": "./src/index.js",
   "author": {
     "name": "Eric Cornelissen",
     "email": "ericornelissen@gmail.com"
   },
   "scripts": {
-    "build": "ncc build src/index.js -m -o lib",
-    "postbuild": "mv lib/index.js lib/index.cjs",
+    "build": "del-cli lib && ncc build src/index.js -m -o lib && node scripts/fix-distribution-file.js",
+    "postbuild": "mv lib/index.js lib/index.mjs",
     "format": "prettier --write .",
     "lint": "prettier --check .",
     "postinstall": "is-ci || husky install",
-    "test": "node --experimental-vm-modules node_modules/.bin/jest"
+    "test": "NODE_OPTIONS=--experimental-vm-modules npx jest"
```

I think that this should be executed with `jest` instead of `npx jest`. The problem solved by indicating the full path with `node_modules/.bin/jest` must be adressed by the user by not doing strange NPM installations, typically adding `./node_modules/.bin` to PATH environment variable.
```suggestion
    "test": "NODE_OPTIONS=--experimental-vm-modules jest"
```

---

I've updated it to `node --experimental-vm-modules node_modules/jest/bin/jest.js`.

Here is the documentation: https://jestjs.io/docs/ecmascript-modules.